pkg/util/validation/user_limits_handler.go

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+// Provenance-includes-location: https://github.com/cortexproject/cortex/blob/master/pkg/util/validation/user_limits_exposition.go
+// Provenance-includes-license: Apache-2.0
+// Provenance-includes-copyright: The Cortex Authors.
+
+package validation
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/grafana/dskit/tenant"
+
+	"github.com/grafana/mimir/pkg/util"
+)
+
+type UserLimitsResponse struct {
+	CompactorBlocksRetentionPeriod float64 `json:"compactor_blocks_retention_period_second"` // suffix with second to make it explicit the value is in second
```

Comment:
In seconds this value will be quite big, and most people configure this in days/months. If we export this value in days, then values like `7.25` still make sense (= 7h4h). (If you prefer seconds, then let's use `int64` instead).

Reply:
I would prefer seconds. With this type, the data is directly usable in Grafana